docs/reference/events.md

```diff
@@ -782,6 +782,11 @@ Default is `false`.
 
 </attribute>
 
+<attribute>
+  <attribute name="no_emails" type="boolean" :parentNames="['settings']" isChild=true>
+    Disables all emails of this event if set true.
+</attribute>
```

Comment:
have you defined the parent 'settings' ? I'm not sure but I'd think it would be necessary

Reply:
@TatianaLekhatkova have you checked this point ?